src/main/resources/liquibase/crop_master.xml

```diff
@@ -83,6 +83,7 @@
 	<include file="crop_changelog/19_3_0.xml" relativeToChangelogFile="true"/>
 	<include file="crop_changelog/19_4_0.xml" relativeToChangelogFile="true"/>
 	<include file="crop_changelog/19_4_3.xml" relativeToChangelogFile="true"/>
+	<include file="crop_changelog/19_5_0.xml" relativeToChangelogFile="true"/>
```

Comment:
@lik-83 a small detail, if this passes branch testing we will merge it as part of 19.4.3
so we will have to move it to that xml file.

Reply:
Done